source/diagrams/manage-diagrams/delete-diagrams.txt

```diff
@@ -1,13 +1,71 @@
 .. _rm-delete-diagrams:
 
-===============
-Delete Diagrams
-===============
+================
+Delete a Diagram
+================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Delete diagrams.
\ No newline at end of file
+You can delete diagrams on the :guilabel:`Mapping` screen of Relational
```

Comment:
[suggestion] We're in the RM docs, so I don't think we have to specify this.

```suggestion
You can delete diagrams on the :guilabel:`Mapping` screen.
```

Reply:
Done!